data_transfer/devices/dreem.py

```diff
@@ -229,7 +229,7 @@ def download_file(self, mongo_id: str) -> None:
         downloaded_file = Path(
             record.download_folder() / f"{record.manufacturer_ref}.h5"
         )
-        record.meta = dict(filesize=downloaded_file.stat().st_size)
+        record.meta["filesize"] = downloaded_file.stat().st_size
```

Comment:
I'm not there with testing (sorry, taking a long time to create all records), but given that `meta` is optional, it will be `None` by default. Then, accessing `["filesize"]` might throw an error. I'll test in a bit, but if so - we need to update the `Record`:

```python
class Record(BaseModel):
    ...

    # Relevant to individual device pipelines
    meta: {}      # default. Indicates type and allows to be empty

    ...
```